bus/bus.go

```diff
@@ -748,16 +753,21 @@ func (b *Bus) renewContractV1(ctx context.Context, cs consensus.State, gp api.Go
 	// derive the renter key
 	renterKey := b.masterKey.DeriveContractKey(c.HostKey)
 
+	// fetch host
+	host, err := b.store.Host(ctx, c.HostKey)
```

Comment:
I think the handler that calls `renewContractV1` already has the host and can pass it in. Because `renewContractV2` also requires the host.